charts/tidb-cluster/templates/monitor-deployment.yaml

```diff
@@ -47,18 +50,34 @@ spec:
             {{- end }}
     {{- end }}
       initContainers:
-      - name: init-data
-        image: {{ .Values.monitor.prometheus.image }}
-        imagePullPolicy: {{ .Values.monitor.prometheus.imagePullPolicy | default "IfNotPresent" }}
+      - name: monitor-initializer
+        image: {{ .Values.monitor.initializer.image }}
+        imagePullPolicy: {{ .Values.monitor.initializer.imagePullPolicy | default "IfNotPresent" }}
+        env:
+        - name: GF_PROVISIONING_PATH
+          value: /grafana-dashboard-definitions/tidb
+        - name: TIDB_CLUSTER_NAME
+          value: {{ template "cluster.name" . }}
+        - name: TIDB_ENABLE_BINLOG
```

Comment:
why add this env? `TIDB_ENABLE_BINLOG `

Reply:
This env is used to decide whether Grafana show BinLog dashboard.